polytope/polytope.py

```diff
@@ -772,21 +775,22 @@ def is_convex(reg, abs_tol=ABS_TOL):
     outer = envelope(reg)
     if is_empty(outer):
         # Probably because input polytopes were so small and ugly..
-        return False,None
+        return False, None
 
-    Pl,Pu = reg.bounding_box
-    Ol,Ou = outer.bounding_box
+    Pl, Pu = reg.bounding_box
+    Ol, Ou = outer.bounding_box
 
-    bboxP = np.hstack([Pl,Pu])
-    bboxO = np.hstack([Ol,Ou])
+    bboxP = np.hstack([Pl, Pu])
+    bboxO = np.hstack([Ol, Ou])
 
-    if sum(abs(bboxP[:,0] - bboxO[:,0]) > abs_tol) > 0 or \
-    sum(abs(bboxP[:,1] - bboxO[:,1]) > abs_tol) > 0:
-        return False,None
+    if sum(abs(bboxP[:, 0] - bboxO[:, 0]) > abs_tol) > 0 or \
```

Comment:
I agree with using parentheses to create a multi-line expression.

[PEP8 argues](https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator) in favor of a line break before the binary operator `or`. For some reason, I have a recollection of reading the opposite convention in PEP8, so I have been following that all this time. The argument I had in mind in favor of breaking the line after the operator is that it prepares the reader for what will come on the next line, thus smoothening the mental blank that the line break introduces while reading (the line break itself serves as a comma). On the other hand, there is the TLA syntax for vertical lists of conjuncts and disjuncts. In any case, I am fine with the `or` at the start of the line.